### PR TITLE
Berkley Package Verification Documentation Upgrade

### PR DESCRIPTION
**Pull request by svv232**
Updating package verification documentation to include some additional environment configuration steps and mina-legacy-create-genesis instructions after stable name change.

### CODE REVIEW

**Review by dkijania — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

File: docs/upgrading-to-berkeley.md [hunk 1]
```diff
@@ -63,10 +63,60 @@ done
...
+            gsutil cp gs://mina_network_block_data/mainnet-<some-version>.json ./block_data.json
+            export PRECOMPUTED_FORK_BLOCK=./block_data.json
+            ```
+    - **Verify Additional Variables:**
+        - Confirm the following variables are properly set:
+            ```bash
+            CONFIG_JSON_GZ_URL=<url to fork config zip> ex: "https://storage.googleapis.com/fork-config-dryrun.json.gz"
+            GENESIS_TIMESTAMP=<same ts as your genesis config> ex: "2024-05-03T00:00:00Z"
+            NETWORK_NAME=<this should match the intended network> ex: mainnet
+            PRECOMPUTED_BLOCK_GS_PREFIX=<url to the network block datta> ex: "gs://mina_network_block_data/mainnet"
```
Comment (deepthiskumar) [inline docs/upgrading-to-berkeley.md:110]: typo- `data`

Comment (svv232) [reply docs/upgrading-to-berkeley.md]: fixed

**Comment by svv232** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**